### PR TITLE
luminous: os/bluestore: recalc_allocated() when decoding bluefs_fnode_t

### PR DESCRIPTION
**Pull request by tchaikov**
http://tracker.ceph.com/issues/23256

### CODE REVIEW

**Comment by tchaikov** [comment]
retest this please.

**Comment by tchaikov** [comment]
retest this please

**Comment by smithfarm** [comment]
jenkins test docs

File: src/os/bluestore/BlueFS.cc [hunk 1]
```diff
@@ -781,6 +780,7 @@ int BlueFS::_replay(bool noop)
         {
 	  bluefs_fnode_t fnode;
 	  ::decode(fnode, p);
+	  fnode.recalc_allocated();
```
Comment (ifed01) [inline src/os/bluestore/BlueFS.cc]: no need after the second commit is applied.

**Comment by tchaikov** [comment]
@ifed01 thanks. fixed and repushed.

**Comment by yuriw** [comment]
[wip-yuri5-testing-2018-03-27-1535-luminous](https://trello.com/c/0zxXqm9m/281-wip-yuri5-testing-2018-03-27-1535-luminous)

**Pull request merged**